addons/website_forum/models/forum.py

```diff
@@ -172,7 +172,7 @@ def _compute_relevancy(self):
     # vote
     vote_ids = fields.One2many('forum.post.vote', 'post_id', string='Votes')
     user_vote = fields.Integer('My Vote', compute='_get_user_vote')
-    vote_count = fields.Integer('Votes', compute='_get_vote_count', store=True)
+    vote_count = fields.Integer('Votes', compute='_get_vote_count', store=True, default=0)
 
```

Comment:
No need to set default=0 here.actually issue is on function _get_vote_count. currently when you didnot get value it's set None, you have to set 0 from function value.